src/index.html

```diff
@@ -11,7 +11,169 @@
+           autocomplete="off"
+          >
+        </label>
+        <label class="form old">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form date">
+          Full date of birth:
+          <input
+           type="date"
+           name="Date"
+           required
+          >
+        </label>
+        <label class="form agreement">
+         I accept the term of the agreement
+          <input
+           type="checkbox"
+           name="accept agreement"
+           required
+          >
+        </label> 
+      </fieldset>
+      <fieldset>
+        <legend>
+         Registration
+        </legend>
+        <label class="form E-mail">
+          E-mail:
+          <input
+           type="email"
+           name="E-mail"
+           class="field"
+           placeholder="email@example.com"
+           required
+          >
+        </label>
+        <label class="form password">
+           Password:
+          <input
+           type="password"
+           name="Password"
+           required
+           minlength="6"
+           maxlength="16"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>
+         An interesting fact about you!
+        </legend>
+         Do you love cats?
+        <label class="form cats">
+          <input
+           type="radio"
+           name="cats"
+           required
+          >
+         Yes
+        </label>
+        <label class="form cats">
+          <input
+            type="radio"
+            name="cats"
+            required
+          >
+         No
+        </label>
+        <label class="form color">
+         What is your favorite color?
+          <input
+           type="color"
+           name="color"
+           value="#000"
+          >
+        </label>
+        <label class="form time">
+         What time do you go to bed?
+          <input
+            type="time"
+            name="time-to-bed"
+            required
+          >
+        </label>
+        <label class="form cars">
+         What are your favorite brand of cars?
+          <select
+           name="cars"
+           multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form range">
+         How do you rate our work?
+          <input
+           type="range"
+           name="work-rate"
+           min="1"
+           max="10"
+           value="1"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>
+         Additional info:
+        </legend>
+        <label class="form comments">
+            Comments:
+          <textarea
+            name="Comments"
+            >
+          </textarea>
```

Comment:
```suggestion
          <textarea name="Comments"></textarea>
```
wrong spaces here, but also here is only one attribute and enough space to do it one-linear :arrow_up: